README.md

```diff
@@ -44,6 +44,11 @@ opt
   * `args` - passed to `is_tuple_expired()` and `process_expired_tuple()` as additional context
   * `tuples_per_iteration` - number of tuples will be checked by one iteration
   * `full_scan_time` - time required for full index scan (in seconds)
+  * `on_iteration_error` - call function on error
+  * `on_iteration_start` - call function on iteration
+  * `on_iteration_complete` - call function on complete iteration
+  * `on_iteration_stop` - call function on end iteration cycle
+  * `max_delay` - delay time between iterations
```

Comment:
I would split `max_delay` to `tuples_per_iteration_delay` and `full_scan_delay`. And I would provide a constant like `expirationd.DELAY_INFINITY = 500 * 365 * 86400 -- 500 years` to pass to `tuples_per_iteration_delay`, when one want to discard the upper limit.

Reply:
Done.

Reply:
expirationd.DELAY_INFINITY doesn't make sense. This delay will hang the fiber forever.